### ISSUE TITLE
OSB 0.3.0 Release

### ISSUE THREAD

**Issue by IanHoang** [opensearch-project/opensearch-benchmark#262]
**Is your feature request related to a problem? Please describe.**

Pushing a new tag for 0.3.0 to add all minor bug fixes to PyPi and Dockerhub/ECR

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by peterzhuamazon** [comment]
Release approval: https://github.com/opensearch-project/opensearch-benchmark/issues/265

**Comment by peterzhuamazon** [comment]
PyPI: https://pypi.org/project/opensearch-benchmark/
Dockerhub: https://hub.docker.com/r/opensearchstaging/opensearch-benchmark/tags?page=1&name=0.3.0

**Comment by IanHoang** [comment]
Dockerhub staging sanity tests:
```
hoangia@3c22fbd0d988 Instance Keys % docker pull opensearchstaging/opensearch-benchmark:0.3.0
0.3.0: Pulling from opensearchstaging/opensearch-benchmark
f1f26f570256: Already exists
2d2b01660885: Already exists
e4e8e4c0b0e1: Already exists
1ba60f086308: Already exists
3c2fccf90be1: Already exists
46a52da095af: Pull complete
46487575520d: Pull complete
11564741a817: Pull complete
4f4fb700ef54: Pull complete
296165b56ea1: Pull complete
Digest: sha256:7f43256a77a238be470cb7835746937b02de64e83c3cd52a2734454ee0c1a97d
Status: Downloaded newer image for opensearchstaging/opensearch-benchmark:0.3.0
docker.io/opensearchstaging/opensearch-benchmark:0.3.0
hoangia@3c22fbd0d988 Instance Keys % docker run opensearchproject/opensearch-benchmark -h
Unable to find image 'opensearchproject/opensearch-benchmark:latest' locally
latest: Pulling from opensearchproject/opensearch-benchmark
f1f26f570256: Already exists
2d2b01660885: Already exists
e4e8e4c0b0e1: Already exists
1ba60f086308: Already exists
3c2fccf90be1: Already exists
68f0af280d3f: Downloading [========================>                          ]  13.86MB/28.51MB
e75c03a65b6e: Download complete
6e76b4c7b50b: Downloading [============================================>      ]  12.89MB/14.57MB
4f4fb700ef54: Download complete
c07b667500cc: Download complete
```
Running help text
```
hoangia@3c22fbd0d988 Instance Keys % docker run opensearchstaging/opensearch-benchmark:0.3.0 -h
usage: opensearch-benchmark [-h] [--version]
                            {execute_test,list,info,create-workload,generate,compare,download,install,start,stop}
                            ...

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

 A benchmarking tool for OpenSearch

options:
  -h, --help            show this help message and exit
  --version             show program's version number and exit

subcommands:
  {execute_test,list,info,create-workload,generate,compare,download,install,start,stop}
    execute_test        Run a benchmark
    list                List configuration options
    info                Show info about a workload
    create-workload     Create a Benchmark workload from existing data
    generate            Generate artifacts
    compare             Compare two test_executions
    download            Downloads an artifact
    install             Installs an OpenSearch node locally
    start               Starts an OpenSearch node locally
    stop                Stops an OpenSearch node locally

Find out more about Benchmark at https://opensearch.org/docs
```
Verify 0.3.0 is latest version
```
$ opensearch-benchmark --version
opensearch-benchmark 0.3.0
```
List workloads
```
docker run opensearchstaging/opensearch-benchmark:0.3.0 list workloads

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

Available workloads:

Name           Description                                                                                                        Documents    Compressed Size    Uncompressed Size    Default TestProcedure         All TestProcedures
-------------  -----------------------------------------------------------------------------------------------------------------  -----------  -----------------  -------------------  ----------------------------  ------------------------------------------------------------------------------------------------------------------------------------------------------------------
geoshape       Shapes from PlanetOSM                                                                                              60,523,283   13.4 GB            45.4 GB              append-no-conflicts           append-no-conflicts
eventdata      This benchmark indexes HTTP access logs generated based sample logs from the elastic.co website using a generator  20,000,000   756.0 MB           15.3 GB              append-no-conflicts           append-no-conflicts,transform
so             Indexing benchmark using up to questions and answers from StackOverflow                                            36,062,278   8.9 GB             33.1 GB              append-no-conflicts           append-no-conflicts
geopoint       Point coordinates from PlanetOSM                                                                                   60,844,404   482.1 MB           2.3 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-fast-with-conflicts
pmc            Full text benchmark with academic papers from PMC                                                                  574,199      5.5 GB             21.7 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-fast-with-conflicts
noaa           Global daily weather measurements from NOAA                                                                        33,659,481   949.4 MB           9.0 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,top_metrics,aggs
http_logs      HTTP server log data                                                                                               247,249,096  1.2 GB             31.1 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-index-only-with-ingest-pipeline,update,append-no-conflicts-index-reindex-only
nested         StackOverflow Q&A stored as nested docs                                                                            11,203,029   663.3 MB           3.4 GB               nested-search-test-procedure  nested-search-test-procedure,index-only
geopointshape  Point coordinates from PlanetOSM indexed as geoshapes                                                              60,844,404   470.8 MB           2.6 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-fast-with-conflicts
nyc_taxis      Taxi rides in New York in 2015                                                                                     165,346,692  4.5 GB             74.3 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts-index-only,update,searchable-snapshot
geonames       POIs from Geonames                                                                                                 11,396,503   252.9 MB           3.3 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-fast-with-conflicts,significant-text
percolator     Percolator benchmark based on AOL queries                                                                          2,000,000    121.1 kB           104.9 MB             append-no-conflicts           append-no-conflicts

--------------------------------
[INFO] SUCCESS (took 12 seconds)
--------------------------------
```
```

**Comment by IanHoang** [comment]
Ran a quick geonames ingest-only test in test-mode
```
$ opensearch-benchmark execute_test --workload=geonames --pipeline=benchmark-only --target-host="<CLUSTER_ENDPOINT>" --client-options="basic_auth_user:'<USERNAME>',basic_auth_password:'<PASSWORD>'" --include-tasks="create-index,index-append" --test-mode

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] You did not provide an explicit timeout in the client options. Assuming default of 10 seconds.
[INFO] Downloading workload data (20.5 kB total size)                             [100.0%]
[INFO] Decompressing workload data from [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json.bz2] to [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Preparing file offset table for [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Executing test with workload [geonames], test_procedure [append-no-conflicts] and provision_config_instance ['external'] with version [1.1.0].

[WARNING] merges_total_time is 11943 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] indexing_total_time is 60489 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] refresh_total_time is 7530 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] flush_total_time is 13045 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
Running create-index                                                           [100% done]
Running index-append                                                           [100% done]

------------------------------------------------------
    _______             __   _____
   / ____(_)___  ____ _/ /  / ___/_________  ________
  / /_  / / __ \/ __ `/ /   \__ \/ ___/ __ \/ ___/ _ \
 / __/ / / / / / /_/ / /   ___/ / /__/ /_/ / /  /  __/
/_/   /_/_/ /_/\__,_/_/   /____/\___/\____/_/   \___/
------------------------------------------------------

|                                                         Metric |         Task |       Value |   Unit |
|---------------------------------------------------------------:|-------------:|------------:|-------:|
|                     Cumulative indexing time of primary shards |              |     1.00928 |    min |
|             Min cumulative indexing time across primary shards |              |           0 |    min |
|          Median cumulative indexing time across primary shards |              |       0.001 |    min |
|             Max cumulative indexing time across primary shards |              |    0.207917 |    min |
|            Cumulative indexing throttle time of primary shards |              |           0 |    min |
|    Min cumulative indexing throttle time across primary shards |              |           0 |    min |
| Median cumulative indexing throttle time across primary shards |              |           0 |    min |
|    Max cumulative indexing throttle time across primary shards |              |           0 |    min |
|                        Cumulative merge time of primary shards |              |     0.19905 |    min |
|                       Cumulative merge count of primary shards |              |          11 |        |
|                Min cumulative merge time across primary shards |              |           0 |    min |
|             Median cumulative merge time across primary shards |              |  0.00156667 |    min |
|                Max cumulative merge time across primary shards |              |   0.0484667 |    min |
|               Cumulative merge throttle time of primary shards |              |           0 |    min |
|       Min cumulative merge throttle time across primary shards |              |           0 |    min |
|    Median cumulative merge throttle time across primary shards |              |           0 |    min |
|       Max cumulative merge throttle time across primary shards |              |           0 |    min |
|                      Cumulative refresh time of primary shards |              |      0.1255 |    min |
|                     Cumulative refresh count of primary shards |              |         384 |        |
|              Min cumulative refresh time across primary shards |              |           0 |    min |
|           Median cumulative refresh time across primary shards |              |  0.00348333 |    min |
|              Max cumulative refresh time across primary shards |              |      0.0362 |    min |
|                        Cumulative flush time of primary shards |              |    0.217417 |    min |
|                       Cumulative flush count of primary shards |              |          72 |        |
|                Min cumulative flush time across primary shards |              |           0 |    min |
|             Median cumulative flush time across primary shards |              | 0.000266667 |    min |
|                Max cumulative flush time across primary shards |              |   0.0461667 |    min |
|                                        Total Young Gen GC time |              |           0 |      s |
|                                       Total Young Gen GC count |              |           0 |        |
|                                          Total Old Gen GC time |              |           0 |      s |
|                                         Total Old Gen GC count |              |           0 |        |
|                                                     Store size |              |    0.179269 |     GB |
|                                                  Translog size |              |   0.0003239 |     GB |
|                                         Heap used for segments |              |    0.323387 |     MB |
|                                       Heap used for doc values |              |   0.0243759 |     MB |
|                                            Heap used for terms |              |    0.246155 |     MB |
|                                            Heap used for norms |              |   0.0290527 |     MB |
|                                           Heap used for points |              |           0 |     MB |
|                                    Heap used for stored fields |              |   0.0238037 |     MB |
|                                                  Segment count |              |          50 |        |
|                                       100th percentile latency | create-index |     5264.26 |     ms |
|                                  100th percentile service time | create-index |     5264.26 |     ms |
|                                                     error rate | create-index |         100 |      % |
|                                                 Min Throughput | index-append |       23.47 | docs/s |
|                                                Mean Throughput | index-append |       23.47 | docs/s |
|                                              Median Throughput | index-append |       23.47 | docs/s |
|                                                 Max Throughput | index-append |       23.47 | docs/s |
|                                        50th percentile latency | index-append |     5353.59 |     ms |
|                                       100th percentile latency | index-append |     5392.48 |     ms |
|                                   50th percentile service time | index-append |     5353.59 |     ms |
|                                  100th percentile service time | index-append |     5392.48 |     ms |
|                                                     error rate | index-append |           0 |      % |

[WARNING] Error rate is 100.0 for operation 'create-index'. Please check the logs.
[WARNING] No throughput metrics available for [create-index]. Likely cause: Error rate is 100.0%. Please check the logs.

--------------------------------
[INFO] SUCCESS (took 57 seconds)
--------------------------------
```

**Comment by peterzhuamazon** [comment]
Production:
* https://gallery.ecr.aws/opensearchproject/opensearch-benchmark
* https://hub.docker.com/r/opensearchproject/opensearch-benchmark/tags?page=1&name=0.3.0

**Comment by IanHoang** [comment]
Prod Dockerhub Sanity Tests:
Pulling image with 0.3.0 on Dockerhub prod
```
docker run --entrypoint bash -it opensearchproject/opensearch-benchmark:0.3.0 -c /bin/sh
Unable to find image 'opensearchproject/opensearch-benchmark:0.3.0' locally
0.3.0: Pulling from opensearchproject/opensearch-benchmark
Digest: sha256:7f43256a77a238be470cb7835746937b02de64e83c3cd52a2734454ee0c1a97d
Status: Downloaded newer image for opensearchproject/opensearch-benchmark:0.3.0
```
Verify version is 0.3.0
```
$ opensearch-benchmark --version
opensearch-benchmark 0.3.0
```
Run list workloads
```
$ opensearch-benchmark list workloads

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

Available workloads:

Name           Description                                                                                                        Documents    Compressed Size    Uncompressed Size    Default TestProcedure         All TestProcedures
-------------  -----------------------------------------------------------------------------------------------------------------  -----------  -----------------  -------------------  ----------------------------  ------------------------------------------------------------------------------------------------------------------------------------------------------------------
geoshape       Shapes from PlanetOSM                                                                                              60,523,283   13.4 GB            45.4 GB              append-no-conflicts           append-no-conflicts
eventdata      This benchmark indexes HTTP access logs generated based sample logs from the elastic.co website using a generator  20,000,000   756.0 MB           15.3 GB              append-no-conflicts           append-no-conflicts,transform
so             Indexing benchmark using up to questions and answers from StackOverflow                                            36,062,278   8.9 GB             33.1 GB              append-no-conflicts           append-no-conflicts
geopoint       Point coordinates from PlanetOSM                                                                                   60,844,404   482.1 MB           2.3 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-fast-with-conflicts
pmc            Full text benchmark with academic papers from PMC                                                                  574,199      5.5 GB             21.7 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-fast-with-conflicts
noaa           Global daily weather measurements from NOAA                                                                        33,659,481   949.4 MB           9.0 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,top_metrics,aggs
http_logs      HTTP server log data                                                                                               247,249,096  1.2 GB             31.1 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-index-only-with-ingest-pipeline,update,append-no-conflicts-index-reindex-only
nested         StackOverflow Q&A stored as nested docs                                                                            11,203,029   663.3 MB           3.4 GB               nested-search-test-procedure  nested-search-test-procedure,index-only
geopointshape  Point coordinates from PlanetOSM indexed as geoshapes                                                              60,844,404   470.8 MB           2.6 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-fast-with-conflicts
nyc_taxis      Taxi rides in New York in 2015                                                                                     165,346,692  4.5 GB             74.3 GB              append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts-index-only,update,searchable-snapshot
geonames       POIs from Geonames                                                                                                 11,396,503   252.9 MB           3.3 GB               append-no-conflicts           append-no-conflicts,append-no-conflicts-index-only,append-sorted-no-conflicts,append-fast-with-conflicts,significant-text
percolator     Percolator benchmark based on AOL queries                                                                          2,000,000    121.1 kB           104.9 MB             append-no-conflicts           append-no-conflicts

--------------------------------
[INFO] SUCCESS (took 11 seconds)
--------------------------------
```
Run geonames test with ingest-only and test-mode
```
$ opensearch-benchmark execute_test --workload=geonames --pipeline=benchmark-only --target-host="https://search-hoangia-aos-testing-2-xpwa7ykgv7bbc7qqfuuzt2no7i.us-east-1.es.amazonaws.com" --client-options="basic_auth_user:'hoangia',basic_auth_password:'Hoangia@123'" --include-tasks="create-index,index-append" --test-mode

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] You did not provide an explicit timeout in the client options. Assuming default of 10 seconds.
[INFO] Downloading workload data (20.5 kB total size)                             [100.0%]
[INFO] Decompressing workload data from [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json.bz2] to [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Preparing file offset table for [/opensearch-benchmark/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Executing test with workload [geonames], test_procedure [append-no-conflicts] and provision_config_instance ['external'] with version [1.1.0].

[WARNING] merges_total_time is 11991 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] indexing_total_time is 60643 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] refresh_total_time is 7536 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
[WARNING] flush_total_time is 13235 ms indicating that the cluster is not in a defined clean state. Recorded index time metrics may be misleading.
Running create-index                                                           [100% done]
Running index-append                                                           [100% done]

------------------------------------------------------
    _______             __   _____
   / ____(_)___  ____ _/ /  / ___/_________  ________
  / /_  / / __ \/ __ `/ /   \__ \/ ___/ __ \/ ___/ _ \
 / __/ / / / / / /_/ / /   ___/ / /__/ /_/ / /  /  __/
/_/   /_/_/ /_/\__,_/_/   /____/\___/\____/_/   \___/
------------------------------------------------------

|                                                         Metric |         Task |       Value |   Unit |
|---------------------------------------------------------------:|-------------:|------------:|-------:|
|                     Cumulative indexing time of primary shards |              |     1.01203 |    min |
|             Min cumulative indexing time across primary shards |              |           0 |    min |
|          Median cumulative indexing time across primary shards |              |       0.001 |    min |
|             Max cumulative indexing time across primary shards |              |    0.208617 |    min |
|            Cumulative indexing throttle time of primary shards |              |           0 |    min |
|    Min cumulative indexing throttle time across primary shards |              |           0 |    min |
| Median cumulative indexing throttle time across primary shards |              |           0 |    min |
|    Max cumulative indexing throttle time across primary shards |              |           0 |    min |
|                        Cumulative merge time of primary shards |              |     0.19985 |    min |
|                       Cumulative merge count of primary shards |              |          12 |        |
|                Min cumulative merge time across primary shards |              |           0 |    min |
|             Median cumulative merge time across primary shards |              |  0.00156667 |    min |
|                Max cumulative merge time across primary shards |              |   0.0492667 |    min |
|               Cumulative merge throttle time of primary shards |              |           0 |    min |
|       Min cumulative merge throttle time across primary shards |              |           0 |    min |
|    Median cumulative merge throttle time across primary shards |              |           0 |    min |
|       Max cumulative merge throttle time across primary shards |              |           0 |    min |
|                      Cumulative refresh time of primary shards |              |      0.1256 |    min |
|                     Cumulative refresh count of primary shards |              |         395 |        |
|              Min cumulative refresh time across primary shards |              |           0 |    min |
|           Median cumulative refresh time across primary shards |              |  0.00348333 |    min |
|              Max cumulative refresh time across primary shards |              |   0.0362167 |    min |
|                        Cumulative flush time of primary shards |              |    0.220583 |    min |
|                       Cumulative flush count of primary shards |              |          77 |        |
|                Min cumulative flush time across primary shards |              |           0 |    min |
|             Median cumulative flush time across primary shards |              | 0.000266667 |    min |
|                Max cumulative flush time across primary shards |              |      0.0469 |    min |
|                                        Total Young Gen GC time |              |           0 |      s |
|                                       Total Young Gen GC count |              |           0 |        |
|                                          Total Old Gen GC time |              |           0 |      s |
|                                         Total Old Gen GC count |              |           0 |        |
|                                                     Store size |              |    0.179388 |     GB |
|                                                  Translog size |              | 0.000646929 |     GB |
|                                         Heap used for segments |              |    0.325634 |     MB |
|                                       Heap used for doc values |              |   0.0331917 |     MB |
|                                            Heap used for terms |              |    0.240753 |     MB |
|                                            Heap used for norms |              |   0.0283203 |     MB |
|                                           Heap used for points |              |           0 |     MB |
|                                    Heap used for stored fields |              |   0.0233688 |     MB |
|                                                  Segment count |              |          49 |        |
|                                       100th percentile latency | create-index |     5220.51 |     ms |
|                                  100th percentile service time | create-index |     5220.51 |     ms |
|                                                     error rate | create-index |         100 |      % |
|                                                 Min Throughput | index-append |       23.56 | docs/s |
|                                                Mean Throughput | index-append |       23.56 | docs/s |
|                                              Median Throughput | index-append |       23.56 | docs/s |
|                                                 Max Throughput | index-append |       23.56 | docs/s |
|                                        50th percentile latency | index-append |     5317.41 |     ms |
|                                       100th percentile latency | index-append |      5343.6 |     ms |
|                                   50th percentile service time | index-append |     5317.41 |     ms |
|                                  100th percentile service time | index-append |      5343.6 |     ms |
|                                                     error rate | index-append |           0 |      % |

[WARNING] Error rate is 100.0 for operation 'create-index'. Please check the logs.
[WARNING] No throughput metrics available for [create-index]. Likely cause: Error rate is 100.0%. Please check the logs.

--------------------------------
[INFO] SUCCESS (took 55 seconds)
--------------------------------
```

**Comment by peterzhuamazon** [comment]
We have officially released the OSB 0.3.0 version on both PyPI and Docker/ECR registries.
